assets/src/dashboard/components/table/index.js

```diff
@@ -35,55 +36,52 @@ export const Table = styled.table`
 export const TableBody = styled.tbody``;
 
 export const TableHeader = styled.thead`
-  background: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray50};
-  border-top: ${({ theme }) => theme.DEPRECATED_THEME.table.border};
-  border-bottom: ${({ theme }) => theme.DEPRECATED_THEME.table.border};
+  background: ${({ theme }) => theme.colors.bg.secondary};
 `;
 
 export const StickyTableHeader = styled(TableHeader)`
   width: 100%;
   border-top: 0;
   border-bottom: 0;
+
   th {
     height: 100%;
-    background: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray50};
-    border-top-width: 0;
-    border-bottom: ${({ theme }) => theme.DEPRECATED_THEME.table.border};
+    background: ${({ theme }) => theme.colors.bg.secondary};
     position: sticky;
-    z-index: ${Z_INDEX.STICKY_TABLE};
+    z-index: 2;
     top: ${THEME_CONSTANTS.WP_ADMIN.TOOLBAR_HEIGHT}px;
+
+    :first-child {
+      border-top-left-radius: ${({ theme }) => theme.borders.radius.small};
+      border-bottom-left-radius: ${({ theme }) => theme.borders.radius.small};
+    }
+
+    :last-child {
+      border-top-right-radius: ${({ theme }) => theme.borders.radius.small};
+      border-bottom-right-radius: ${({ theme }) => theme.borders.radius.small};
+    }
   }
 `;
 
 export const TableHeaderCell = styled.th`
-  padding: ${({ theme }) => theme.DEPRECATED_THEME.table.headerCellPadding}px;
-  font-weight: ${({ theme }) =>
-    theme.DEPRECATED_THEME.typography.weight.normal};
-  color: ${({ theme }) => theme.DEPRECATED_THEME.colors.gray500};
+  padding: 16px;
   text-align: left;
+  white-space: nowrap;
 `;
 
 export const TablePreviewHeaderCell = styled(TableHeaderCell)`
   padding-right: 0;
-
-  @media ${({ theme }) => theme.DEPRECATED_THEME.breakpoint.largeDisplayPhone} {
-    display: none;
-  }
```

Comment:
Removed a lot of these since there's going to be a min width of `1098px`. We'd never reach these sizes.